lib/eventFactory.js

```diff
@@ -424,13 +426,14 @@ function createBuilds(config) {
  *     { "name": "name2", "jobs": [4, 5], "pipelineId": 123, "groupEventId": 555 }]
  * @method convertStages
  * @param  {Object}     config              config
- * @param  {Number}     config.groupEventId Group event ID
+ * @param  {Number}     config.eventId      Event ID
+ * @param  {Object}     config.jobs         Jobs
  * @param  {Object}     config.pipeline     Pipeline
  * @param  {Object}     config.stages       Pipeline stages
  * @return {Array}                New array with stages after up-converting
  */
 async function convertStages(config) {
-    const { pipeline, stages, groupEventId } = config;
+    const { pipeline, stages, eventId, jobs } = config;
```

Comment:
`eventId` is not a valid field in stage model.
```suggestion
    const { pipeline, stages, jobs } = config;
```